ouroboros-consensus/src/Ouroboros/Consensus/Node/ProtocolInfo.hs

```diff
@@ -26,8 +26,8 @@ protocolInfo :: NumCoreNodes
              -> Protocol blk
              -> ProtocolInfo blk
 protocolInfo nodes nid demoProtocol = case demoProtocol of
-    ProtocolMockBFT        params          -> protocolInfoBft       nodes nid params
-    ProtocolMockPraos      params          -> protocolInfoPraos     nodes nid params
-    ProtocolLeaderSchedule params schedule -> protocolInfoPraosRule nodes nid params schedule
-    ProtocolMockPBFT       params          -> protocolInfoMockPBFT  nodes nid params
-    ProtocolRealPBFT       params gc mplc  -> protocolInfoByron     nodes nid params gc mplc
+    ProtocolMockBFT        params               -> protocolInfoBft       nodes nid params
+    ProtocolMockPraos      params               -> protocolInfoPraos     nodes nid params
+    ProtocolLeaderSchedule params schedule      -> protocolInfoPraosRule nodes nid params schedule
+    ProtocolMockPBFT       params               -> protocolInfoMockPBFT  nodes nid params
+    ProtocolRealPBFT       gc mthr prv swv mplc -> protocolInfoByron     gc mthr prv swv mplc
```

Comment:
Bikeshedding, but changes like this are why I don't like styles which align identifiers. There's only one change here, but it looks like a whole block has changed.